modules/ROOT/pages/composite-databases/administration.adoc

```diff
@@ -22,19 +22,27 @@ Constituent graphs are added with the link:{neo4j-docs-base-uri}/cypher-manual/{
 ====
 [source, cypher]
 ----
-CREATE ALIAS cineasts.latest
+CREATE ALIAS `cineasts`.`latest`
   FOR DATABASE movies2022
 ----
 ====
 
+[NOTE]
+====
+`.` is used as a separating character between the namespace and database name in aliases.
+It is therefore recommended to avoid `.` characters in database and composite database names.
+If '.' is needed in either name, the name must be surrounded by backticks, i.e. `composite.`db.name``. Backticks may be added regardless of whether the name contains special characters or not, so it is good practice to always backtick both names, i.e. `pass:[`composite`.`db.name`]`.
+Another alternative is to use query parameters.
```

Comment:
parameters and dots are 'fun' XD

Reply:
Removed this "solution" since it aparently isn't a solution 🤦